python/lsst/ip/isr/overscan.py

```diff
@@ -197,14 +204,26 @@ def run(self, exposure, amp, isTransposed=False):
             parallelOverscanBBox = amp.getRawParallelOverscanBBox()
             imageBBox = amp.getRawDataBBox()
 
-            parallelResults = self.correctOverscan(exposure, amp,
-                                                   imageBBox, parallelOverscanBBox,
-                                                   isTransposed=not isTransposed)
-
-            overscanMean = (overscanMean, parallelResults.overscanMean)
-            overscanSigma = (overscanSigma, parallelResults.overscanSigma)
-            residualMean = (residualMean, parallelResults.overscanMeanResidual)
-            residualSigma = (residualSigma, parallelResults.overscanSigmaResidual)
+            maskIm = exposure.getMaskedImage()
+            maskIm = maskIm.Factory(maskIm, parallelOverscanBBox)
+            makeThresholdMask(maskIm, threshold=self.config.numSigmaClip, growFootprints=0)
+            maskPix = countMaskedPixels(maskIm, self.config.maskPlanes)
+            xSize, ySize = parallelOverscanBBox.getDimensions()
+            if maskPix > xSize*ySize*self.config.maskPixFracThresh:
+                self.log.warning('Fraction of masked pixels for parallel overscan calculation larger'
+                                 ' than %f of total pixels (i.e. %f masked pixels) on amp %s.',
+                                 self.config.maskPixFracThresh, maskPix, amp.getName())
+                self.log.warning('Not doing parallel overscan correction.')
+                parallelResults = pipeBase.Struct()
```

Comment:
This `parallelResults` isn't used at all, so is unnecessary.  But I'm wondering if `overscanMean` should be a tuple `(overscanMean, 0.0)` or it may be fine to not have the second part of the tuple because the parallel overscan wasn't run at all.